explainaboard/processors/sequence_labeling.py

```diff
@@ -0,0 +1,573 @@
+from __future__ import annotations
```

Comment:
There are lots of places in this file that refer to "entity" (e.g. in the feature descriptions). This should probably be generalized to "span".

Reply:
Thanks! Will clear out all `entity`s. 

>"I think that what you're calling "sequence labeling" here is actually something more like "span prediction"?"
Good point. I also hesitate among `tagging`, `sequence labeling`, `span prediction`.  Let me mark it as an issue.